### PR TITLE
Release fixing observer mode for Amazon + BillingClient 3 

### PR DESCRIPTION
**Pull request by vegaro**
- Skip registering listener in Amazon observer mode
     https://github.com/RevenueCat/purchases-android/pull/478
- Add new function (`syncObserverModeAmazonPurchase`) to sync a single Amazon purchase with our backend
     https://github.com/RevenueCat/purchases-android/pull/470
- Add DangerousSettings to configure function so automatic syncing of purchases can be disabled. 
  Not recommended unless instructed by RevenueCat support.
     https://github.com/RevenueCat/purchases-android/pull/451

### CODE REVIEW

File: common/src/main/java/com/revenuecat/purchases/common/Config.kt [hunk 1]
```diff
@@ -4,5 +4,5 @@ object Config {
 
     var debugLogsEnabled = false
 
-    const val frameworkVersion = "5.0.0-amazon.alpha.3.billing3.iap2"
+    const val frameworkVersion = "5.0.0-amazon.alpha.5.billing3.iap2"
```
Comment (vegaro) [inline common/src/main/java/com/revenuecat/purchases/common/Config.kt:7]: Increasing to 5 to match purchases-hybrid-common and make it easier to track

File: .circleci/config.yml [hunk 2]
```diff
@@ -337,17 +319,4 @@ workflows:
             branches:
               only:
                 - main
-      - hold:
```
Comment (beylmk) [inline .circleci/config.yml:340]: phew

File: fastlane/README.md [hunk 3]
```diff
@@ -56,6 +56,11 @@ Upload a snapshot release
 fastlane android prepare_next_version
 ```
 Prepare next version
+### android deploy_amazon
+```
+fastlane android deploy_amazon
```
Comment (beylmk) [inline fastlane/README.md:61]: this won't work since we deleted it right?

**Review by beylmk — APPROVED** [review]
🥳

File: fastlane/README.md (hunk 3 above)
Comment (vegaro) [reply fastlane/README.md:61]: oh right, just a README tho, I can delete it later

**Pull request merged**